### PR TITLE
Simplify required fields

### PR DESCRIPTION
**Pull request by alanmoo**
Closes https://github.com/mozilla/network-pulse-api/issues/22

How are migrations handled when a server is already live? Do I need to just `makemigration` starting from a database that's identical to the one that's on the server and then commit the migration files?

### CODE REVIEW

File: pulseapi/entries/models.py [hunk 1]
```diff
@@ -23,17 +23,17 @@ class Entry(models.Model):
     A pulse entry
     """
     title = models.CharField(max_length=140)
-    description = models.CharField(max_length=300)
-    content_url = models.URLField()
-    thumbnail_url = models.URLField()
+    description = models.CharField(max_length=300, blank=True)
+    content_url = models.URLField(blank=True)
```
Comment (Pomax) [inline pulseapi/entries/models.py]: Are we sure about this one? Are people supposed to be able to post entries without a URL to point to the thing the entry was supposed to be for?

Comment (alanmoo) [inline pulseapi/entries/models.py]: You're right, I goofed. Fixed now.

File: pulseapi/entries/test_views.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def test_post_minimum_entry(self):
         """
         Test posting an entry with minimum amount of content
         """
-        payload = self.generatePostPayload(data={'title':'title test_post_minimum_entry'})
+        payload = self.generatePostPayload(data={'title':'title test_post_minimum_entry', 'tags':''})
```
Comment (Pomax) [inline pulseapi/entries/test_views.py]: We probably also want to make sure we have full test coverage for required fields now. https://github.com/mozilla/network-pulse-api/blob/fb8a57cff6fcd027bf9eb9766a71bd1ae5d7733f/pulseapi/entries/test_views.py#L38 tests for title, but with `content_url` also being required, we want to make sure that's a 400 as well?

Comment (alanmoo) [inline pulseapi/entries/test_views.py]: Added.

File: pulseapi/entries/test_views.py [hunk 3]
```diff
@@ -36,23 +36,18 @@ def test_post_duplicate_title(self):
         self.assertEqual(len(entriesJson), 4)
 
     def test_post_empty_title(self):
-        """Make sure entries require a title"""
+        """Make sure entries require a title and content_url"""
```
Comment (Pomax) [inline pulseapi/entries/test_views.py]: can we split this up into two tests?

**Pull request merged**